java/src/main/native/src/RmmJni.cpp

```diff
@@ -20,6 +20,7 @@
 #include <iostream>
 #include <limits>
 
+#include <cudf/utilities/error.hpp>
```

Comment:
Is this really necessary?  It's the only modification to this file, and implies some header should be including this instead to be self-contained.

Reply:
Removed